website/content/en/preview/concepts/setup.md

```diff
@@ -0,0 +1,554 @@
+              "Resource": "arn:${AWS::Partition}:eks:${AWS::Region}:${AWS::AccountId}:cluster/${ClusterName}",
+              "Action": "eks:DescribeCluster"
+            }
+          ]
+        }
+```
+
+
+
+# Interruption Handling 
+Settings in this section allow the Karpenter controller to interact with interruption queues.
+So, for example, if Spot instances are being reclaimed or a node crashes, seeing messages from these queues allows Karpenter to be proactive in moving workloads or adding new nodes.
+Defining the `KarpenterInterruptionQueuePolicy` allows Karpenter to see and respond to the following:
+
+* AWS health events
+* Spot interruptions
+* Spot rebalance recommendations
+* Instance state changes
+
+The resources defined in this section include:
+
+* KarpenterInterruptionQueue
+* KarpenterInterruptionQueuePolicy
+* ScheduledChangeRule
+* SpotInterruptionRule
+* RebalanceRule
+* InstanceStateChangeRule
+
+## KarpenterInterruptionQueue
+The [AWS::SQS::Queue](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sqs-queue.html) resource is used to create an Amazon SQS standard queue.
+Properties of that resource set the `QueueName` to the name of your cluster, the time for which SQS retains each message (`MessageRetentionPeriod`) to 300 seconds, and enabling serverside-side encryption using SQS owned encryption keys (`SqsManagedSseEnabled`) to `true`.
+See [SetQueueAttributes](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SetQueueAttributes.html) for descriptions of some of these attributes.
+
+```
+  KarpenterInterruptionQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub "${ClusterName}"
+      MessageRetentionPeriod: 300
+      SqsManagedSseEnabled: true
+```
+
+## KarpenterInterruptionQueuePolicy
+The Karpenter interruption queue policy is created and applied to allow the Karpenter controller to see messages for selected services.
+In particular, the [AWS::SQS::QueuePolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sqs-queuepolicy.html) resource here applies `EC2InterruptionPolicy` to the `KarpenterInterruptionQueue`. The policy allows [sqs:SendMessage](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SendMessage.html) actions to `events.amazonaws.com` and `sqs.amazonaws.com` services. It also allows the `GetAtt` function to get attributes from `KarpenterInterruptionQueue.Arn`.
+
+```
+  KarpenterInterruptionQueuePolicy:
+    Type: AWS::SQS::QueuePolicy
+    Properties:
+      Queues:
+        - !Ref KarpenterInterruptionQueue
+      PolicyDocument:
+        Id: EC2InterruptionPolicy
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - events.amazonaws.com
+                - sqs.amazonaws.com
+            Action: sqs:SendMessage
+            Resource: !GetAtt KarpenterInterruptionQueue.Arn
+```
+
+## ScheduledChangeRule
```

Comment:
I would consider creating a "rules" subsection that then defines each of these rules and what the purpose for each of these rules is.

Reply:
Okay. I changed the smaller rules sections into a single "Rules" section and made each on into a bullet item in that section.